RELEASE-NOTES.txt

```diff
@@ -4,7 +4,8 @@
 
 4.41
 -----
-
+-   Some bug fixes #1359
+-   Migrated core data database to secure shared directory #1349
```

Comment:
I think we should mark this as internal, because end users wouldn't know what this means. Alternatively, we can make this more end user friendly, if you think that's possible.